apidoc/Titanium/UI/AlertDialog.yml

```diff
@@ -199,6 +199,14 @@ properties:
     type: Number
     default: undefined (Android), -1 (iOS)
 
+  - name: buttonPressRequired
+    summary: |
+        This property determines whether the dialog can be cancelled by clicking the System Back button on Android.
+    type: Boolean
+    default: true on Android
+    since: "7.4.0"
+    platforms: [android]
```

Comment:
Default value should be `false` to retain the old behavior.

And how about we change the summary to...
> Setting this to true requires the end-user to click a dialog button to close the dialog.

And perhaps the following description...
> Set to true to prevent the dialog from being dismissed via back navigation or tapping outside of the dialog. This requires the end-user to click on one of the dialog buttons provided by property buttonNames. Note that if the dialog does not have any buttons, then the dialog can only be closed programmatically via the hide() method.